README.md

```diff
@@ -44,17 +44,17 @@ If you don't want to use the provided connection method you can follow a guide o
 
 ### Submitting blocks
 
-Currently relays accept blocks as either JSON or SSZ encoded bytes of `capella.SubmitBlockRequest` from this module `github.com/attestantio/go-builder-client/api/capella`, which means most builders are already generating this paylaod when submitting blocks today.
+Currently relays accept blocks as either JSON or SSZ encoded bytes of `deneb.SubmitBlockRequest` from this module `github.com/attestantio/go-builder-client/api/deneb`, which means most builders are already generating this paylaod when submitting blocks today.
 
-The provided utility function `CapellaRequestToProtoRequest` accepts `capella.SubmitBlockRequest` and returns `*grpc.SubmitBlockRequest` which can then be sent to the channel returned by `grpc.NewConnection`.
+The provided utility function `DenebRequestToProtoRequest` accepts `deneb.SubmitBlockRequest` and returns `*grpc.SubmitBlockRequest` which can then be sent to the channel returned by `grpc.NewConnection`.
```

Comment:
may want to update readme here and in surrounding lines to use `VersionedRequestToProtoRequest()` and `"github.com/attestantio/go-builder-client/spec"  VersionedSubmitBlockRequest` instead